api/python/provisioner/commands/deploy_vm.py

```diff
@@ -194,25 +241,29 @@ def run(self, **kwargs):
                     pillar = PillarResolver(local_minion_id()).get(
                         [metadata_device_keypath]
                     )
-                    metadata_device = pillar[local_minion_id()][0]
+                    metadata_device = pillar[local_minion_id()][metadata_device_keypath][0]  # noqa: E501
                     metadata_device = f"{metadata_device}1"
                     # TODO IMPROVE EOS-12076 hard coded
-                    mount_point = '/var/mero'
+                    mount_point = '/var/motr'
 
                     logger.info(
                         f"Mounting partition {metadata_device} "
-                        "into {mount_point} (with fstab record)"
+                        f"into {mount_point} (with fstab record)"
                     )
                     StateFunExecuter.execute(
                         'mount.mounted',
-                        fun_args=[mount_point, metadata_device, 'ext4'],
-                        fun_kwargs=dict(mkmnt=True, persist=True)
+                        fun_kwargs=dict(
```

Comment:
please check my comment above

Reply:
comment is addressed

Reply:
that won't work for remote setup, try the following instead:
```
               fun_kwargs=dict(...)

               # TODO IMPROVE move that conditional split into Deploy class
               if self.setup_ctx:
                   return self.setup_ctx.ssh_client.run(
                        'state.single',
                        fun_args=['mount.mounted'],
                        fun_kwargs=fun_kwargs,
                    )
               else:
                 StateFunExecuter.execute(
                        'mount.mounted',
                        fun_kwargs=fun_kwargs
                )   
```

Reply:
comment is addressed